### PR TITLE
Isso comments plugin

### PR DESCRIPTION
**Pull request by ArthurHoaro**
Use Isso client to let visitors comments on permalinks

Note: also affect private links because links can previously commented links can be set to private and the other way around.

Note bis: It's not beautifully integrated for now.

ping #181

### CODE REVIEW

**Comment by nodiscc** [comment]
Hey @ArthurHoaro , looks cool. Just to make sure I understand (I have not tested it yet):

```php
if (count($data['links']) == 1) { 
```

The comments box is only displayed when there is only one link visible (access by permalink, or search with a single result)? Or does it litter the linklist with 30 comment boxes? The first solution is preferable.

Simple code, great work

**Comment by ArthurHoaro** [comment]
Thanks! Yes, it is displayed only when 1 link is displayed. However, I didn't think of search results. It probably shouldn't be displayed in this case.

**Comment by kalvn** [comment]
To avoid ISSO to be displayed on search result, you could use the same condition I used for my [disqus plugin](https://github.com/kalvn/shaarli-plugin-disqus): checking that there is only 1 result and that the query string parameter `page` is not defined (since it's not present on permalink pages).

```php
if(count($data['links']) === 1 && !isset($_GET['page']) && !empty($disqusShortname)){
```

Just ignore the last part of the condition, here.

**Comment by ArthurHoaro** [comment]
Updated! 

  * Only displayed on permalinks. I did something similar @kalvn, but without requiring to access `$_GET` within the plugin.
  * Unit tests.
  * One line of CSS to, at least, align the title.

File: plugins/isso/isso.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#isso-thread > h4 {
+    text-align: center;
+}
```
Comment (virtualtam) [inline plugins/isso/isso.css]: newline

File: plugins/isso/isso.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * Plugin Isso.
+ */
+
+$issoUrl = $conf->get('plugins.ISSO_SERVER');
```
Comment (virtualtam) [inline plugins/isso/isso.php]: access to `$conf` could be made directly from the Linklist render hook, which would help keeping test code clean

Comment (ArthurHoaro) [inline plugins/isso/isso.php]: I'll use #659. The errors were assigned at PageBuilder initialization, which prevent from using your solution.

**Comment by ArthurHoaro** [comment]
Updated.

**Pull request merged**